tests/integration/kubernetes/confidential_common.sh

```diff
@@ -82,3 +82,28 @@ function is_confidential_hardware() {
 
 	return 1
 }
+
+function create_loop_device(){
+	local loop_file="${1:-/tmp/trusted-storage.img}"
+	cleanup_loop_device
+
+	sudo dd if=/dev/zero of=$loop_file bs=1M count=2500
+	sudo losetup -fP $loop_file >/dev/null 2>&1
+	local device=$(sudo losetup -j $loop_file | awk -F'[: ]' '{print $1}')
```

Comment:
You have to use `exec_host()` here, otherwise the loop device will be created on the runner machine, rather than the host node.

Reply:
Thanks for your comment @sprt. Let's change the code and see if the test passes on coco-dev CI.